FeedSX/screens/UniversalFeed/index.tsx

```diff
@@ -189,7 +265,50 @@ const UniversalFeed = () => {
   };
 
   return (
-    <View style={{height: '100%'}}>
+    <SafeAreaView style={{height: '100%'}}>
+      {/* header */}
+      <LMHeader heading={APP_TITLE} />
+      {/* post uploading section */}
+      {postUploading && (
+        <View
+          style={styles.postUploadingView}>
+          <View style={{flexDirection: 'row', alignItems: 'center'}}>
+            {/* post uploading media preview */}
+            {uploadingMediaAttachmentType === IMAGE_ATTACHMENT_TYPE && (
+              <LMImage
+                imageUrl={uploadingMediaAttachment}
+                imageStyle={{backgroundColor: '#fff'}}
+                boxStyle={styles.uploadingImageVideoBox}
+                width={49}
+                height={42}
+              />
+            )}
+            {uploadingMediaAttachmentType === VIDEO_ATTACHMENT_TYPE && (
+              <LMVideo
+                videoUrl={uploadingMediaAttachment}
+                videoStyle={{backgroundColor: '#fff'}}
+                boxStyle={styles.uploadingImageVideoBox}
+                width={49}
+                height={42}
+                showControls={false}
+                boxFit="contain"
+              />
+            )}
+            {uploadingMediaAttachmentType === DOCUMENT_ATTACHMENT_TYPE && (
+              <LMIcon
+                assetPath={require('../../assets/images/pdf_icon3x.png')}
+                type="png"
+                iconStyle={{marginRight: 2, resizeMode: 'contain'}}
+                height={40}
+                width={45}
```

Comment:
use height and from a constant variable if you are reusing it. Inside styles.ts constant file add width, height styles for icon